framework/include/problems/FEProblemBase.h

```diff
@@ -134,6 +135,19 @@ enum class MooseLinearConvergenceReason
   DIVERGED_PCSETUP_FAILED = -11
 };
 
+enum class MooseCurrentErrand
```

Comment:
Task and Job already have different meanings. The Thesaurus gave me "errand". If anyone has a better term (maybe _stage_?) let me know.